src/main/search-engine.ts

```diff
@@ -180,6 +181,10 @@ export class SearchEngine {
         return this.favoriteManager.clearExecutionLog();
     }
 
+    public clearCommandlineHistory(): Promise<void> {
+        return new ElectronStoreCommandlineHistoryRepository().clearAll();
+    }
```

Comment:
The SearchEngine should also not care about specific plugins, because it does not know which specific plugins are in use. Also not a very scalable solution.

Reply:
This means I will have to remove the option to clear the plugin history.